<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receptionist</class>
 <widget class="QMainWindow" name="Receptionist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radioButton_checkout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>办理退房</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_checkin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>办理入住</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_yes">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_receptionistID_tag">
       <property name="text">
        <string>前台服务员：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_receptionistID">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>189</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_roomID_tag">
       <property name="text">
        <string>房间号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_roomID"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_bill_tag">
       <property name="text">
        <string>账单：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser_bill"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
